apis/core/lib/domain/column-mapping/create.ts

```diff
@@ -0,0 +1,34 @@
+import { GraphPointer } from 'clownface'
+import { rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { ResourceStore } from '../../ResourceStore'
+import * as id from '../identifiers'
+import { resourceStore } from '../resources'
+import { NamedNode } from 'rdf-js'
+
+interface CreateColumnMappingCommand {
+  table: GraphPointer<NamedNode>
+  resource: GraphPointer
+  store?: ResourceStore
+}
+
+export async function createColumnMapping({
+  table,
+  resource,
+  store = resourceStore(),
+}: CreateColumnMappingCommand): Promise<GraphPointer> {
+  const label = resource.out(schema.label).term!
+  const columnMapping = id.columnMapping(table, label.value)
+
+  const graph = table.out(columnMapping)
```

Comment:
`ColumnMapping` should be a resource. Check  https://github.com/zazuko/cube-creator/blob/master/apis/core/lib/domain/table/create.ts#L78, I already started a function to create a column mapping